Automattic/wp-calypso
# Fix the new domain warning links to point to the proper support doc

#### Changes proposed in this Pull Request

* The "Learn more" links in the warning that is shown when you register a new domain point to a generic domain support document. This fixes those to point to the relevant section.
* The "Learn more" links in the Set primary domain screen should point to the relevant section on primary domains in the support docs

#### Testing instructions

* Register a new domain and verify where the Learn more links point to
* Click on domain setting -> Set as primary button and verify that the Learn more link points to the correct section (you need to disable the new domain status screens design with the following query string:
`?flags=-domains/new-status-design`

<img width="745" alt="Screenshot 2020-03-01 at 21 47 09" src="https://user-images.githubusercontent.com/1355045/75632598-4f680e00-5c06-11ea-882b-98ee903b36c9.png">
